bundle/edu.gemini.phase2.skeleton.servlet/src/main/resources/edu/gemini/phase2/template/factory/xml/GRACES_BP.xml

```diff
@@ -243,15 +244,11 @@ Central wavelength: XXXXA / XXXnm
             <param name="exposureTime" value="60.0"/>
             <param name="posAngle" value="0"/>
             <param name="coadds" value="1"/>
-            <paramset name="parallacticAngleDuration">
-              <param name="parallacticAngleDurationMode" value="REMAINING_TIME"/>
-              <param name="explicitDuration" value="0"/>
-            </paramset>
             <param name="adc" value="NONE"/>
-            <param name="ampCount" value="SIX"/>
+            <param name="ampCount" value="TWELVE"/>
             <param name="gainChoice" value="LOW"/>
             <param name="ampReadMode" value="FAST"/>
-            <param name="detectorManufacturer" value="E2V"/>
+            <param name="detectorManufacturer" value="HAMAMATSU"/>
```

Comment:
Does this require a model change?

Reply:
No, there is already an enumeration with those two values.